src/lib/stitching/exchange/transformers/replaceCommerceDateTimeType.ts

```diff
@@ -72,14 +75,18 @@ export class ReplaceCommerceDateTimeType implements Transform {
     Object.entries(fields).forEach(([fieldName, fieldDefinition]) => {
       const fieldConfig = fieldToFieldConfig(fieldDefinition, resolveType, true)
       // If it's not a type we want to replace, just skip it
-      if (fieldDefinition.type.name !== "CommerceDateTime") {
-        newFields[fieldName] = fieldConfig
-      } else {
+      if (
+        fieldDefinition.type.name === "CommerceDateTime" ||
+        (fieldDefinition.type.ofType &&
+          fieldDefinition.type.ofType.name === "CommerceDateTime")
```

Comment:
There’s a function in GraphQL-JS to get an unwrapped type (unwrapping nonnull and lists). I believe it’s called `getNamedType` or something like that.

Reply:
Nice! switched to use that for checking non null fields.